src/components/YourCompany/OfficeAddresses.vue

```diff
@@ -383,9 +383,10 @@ export default class OfficeAddresses extends Mixins(CommonMixin) {
       this.originalOfficeAddresses.recordsOffice?.deliveryAddress, ['addressCountryDescription'])
   }
 
-  /** True if any office address has changed. */
+  /** True if any office address has changed. Applies to corrections only. */
   private get officeAddressesChanged (): boolean {
-    return (this.mailingChanged || this.deliveryChanged || this.recMailingChanged || this.recDeliveryChanged)
+    return this.isCorrectionFiling &&
```

Comment:
Because we were not loading up the original IA (ie, as per corrections), then this was always true, which is incorrect.

Reply:
nice catch